app/presenters/migration/index.rb

```diff
@@ -18,17 +18,17 @@ def content
             license_public:   true
           },
           description: t('description'),
-          closing_remark: t('closing_remark'),
           begin_link: {
             url: new_migration_path,
             text: t('begin_link.text'),
             text_long: t('begin_link.text_long')
           },
           call_to_action: call_to_action,
           previews: [
-            preview_data(1),
-            preview_data(2),
-            preview_data(3, is_person: true, button_text: t('preview_3.button_text'))
+            preview_data(1, url: 'https://www.europeana.eu/portal/en/record/2022608/FBIB_FBib_07004_073.html'),
+            preview_data(2, url: 'https://www.europeana.eu/portal/en/record/2021609/objecten_60411_A_B.html'),
+            preview_data(3, is_person: true, button_text: t('preview_3.button_text')),
+            preview_data(4, button_text: t('preview_4.button_text'), url: 'https://historia-europa.ep.eu/sites/all/themes/custom/heh/images/heh-logo-mute.png'),
```

Comment:
Is this URL correct? Why would we want to link to another copy of the same image?

Reply:
No.  I've updated the links